browser/profiles/brave_profile_manager.cc

```diff
@@ -183,7 +183,8 @@ void BraveProfileManager::MigrateProfileNames() {
 
 void BraveProfileManager::OnProfileAdded(Profile* profile) {
   content::URLDataSource::Add(
-      profile,
+      (profile && profile->IsSystemProfile()) ? profile->GetPrimaryOTRProfile()
```

Comment:
cc @darkdh is this ok for Tor?

Reply:
yes, this change is only for system profile

Reply:
there's an OTR system profile?